_shared_assets/themes/owncloud_org/layout.html

```diff
@@ -115,6 +115,22 @@
 <meta name="theme-color" content="#1d2d44">
 <link rel="shortcut icon" href="https://owncloud.org/wp-content/themes/owncloudorgnew/assets/img/common/favicon.png" />
 <link rel="apple-touch-icon-precomposed" href="https://owncloud.org/wp-content/themes/owncloudorgnew/assets/img/common/favicon.png" />
+
+<!-- Piwik -->
+<script type="text/javascript">
+  var _paq = _paq || [];
+  /* tracker methods like "setCustomDimension" should be called before "trackPageView" */
+  _paq.push(['trackPageView']);
+  _paq.push(['enableLinkTracking']);
+  (function() {
+    var u="//piwik.owncloud.com/";
```

Comment:
Same here

Reply:
Ah, here you can even set `var u="https//piwik.owncloud.org/";` just to keep the requests within the same TLD.

Reply:
np. shall do.